### PR TITLE
bybit/buf: Fix test issues

### PR DESCRIPTION
**Pull request by thrasher-**
# PR Description

A minor PR to fix bybit test and buf format issues.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
 
## How has this been tested

- [x] go test ./... -race
- [x] golangci-lint run

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation and regenerated documentation via the documentation tool
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally and on Github Actions/AppVeyor with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: exchanges/bybit/futures_type.go [hunk 1]
```diff
@@ -51,32 +51,32 @@ type FuturesCandleStickWithStringParam struct {
...
+	TotalTurnover          bybitNumericalValue `json:"total_turnover"` // can either be an empty string or a number
+	Turnover24h            float64             `json:"turnover_24h,string"`
+	TotalVolume            float64             `json:"total_volume"`
+	Volume24h              float64             `json:"volume_24h"`
+	FundingRate            float64             `json:"funding_rate,string"`
+	PredictedFundingRate   bybitNumericalValue `json:"predicted_funding_rate"` // can either be an empty string or a number
+	NextFundingTime        string              `json:"next_funding_time"`
+	CountdownHour          int64               `json:"countdown_hour"`
+	DeliveryFeeRate        bybitNumericalValue `json:"delivery_fee_rate"`        // can either be an empty string or a number
+	PredictedDeliveryPrice bybitNumericalValue `json:"predicted_delivery_price"` // can either be an empty string or a number
```
Comment (gloriousCode) [inline exchanges/bybit/futures_type.go:78]: Some more accurate comments
```suggestion
	PricePcntChange1h      bybitNumericalValue `json:"price_1h_pcnt"` // can be an empty or numeric string
	MarkPrice              float64             `json:"mark_price,string"`
	IndexPrice             float64             `json:"index_price,string"`
	OpenInterest           float64             `json:"open_interest"`
	OpenValue              bybitNumericalValue `json:"open_value"`     // can be an empty or numeric string
	TotalTurnover          bybitNumericalValue `json:"total_turnover"` // can be an empty or numeric string
	Turnover24h            float64             `json:"turnover_24h,string"`
	TotalVolume            float64             `json:"total_volume"`
	Volume24h              float64             `json:"volume_24h"`
	FundingRate            float64             `json:"funding_rate,string"`
	PredictedFundingRate   bybitNumericalValue `json:"predicted_funding_rate"` // can be an empty or numeric string
	NextFundingTime        string              `json:"next_funding_time"`
	CountdownHour          int64               `json:"countdown_hour"`
	DeliveryFeeRate        bybitNumericalValue `json:"delivery_fee_rate"`        // can be an empty or numeric string
	PredictedDeliveryPrice bybitNumericalValue `json:"predicted_delivery_price"` // can be an empty or numeric string
```

File: exchanges/bybit/futures_type.go [hunk 2]
```diff
@@ -149,10 +149,11 @@ type OpenInterestData struct {
 
 // BigDealData stores big deal data
 type BigDealData struct {
-	Symbol string `json:"symbol"`
-	Side   string `json:"side"`
-	Time   int64  `json:"timestamp"`
-	Value  int64  `json:"value"`
+	ID     int64   `json:"id"`
```
Comment (gloriousCode) [inline exchanges/bybit/futures_type.go:152]: This change is a big deal

**Review by gloriousCode — CHANGES_REQUESTED** [review]
Thanks! I've tested and appreciated the fixes, but I still would like one minor change :smile:

File: exchanges/bybit/futures_type.go (hunk 2 above)
Comment (thrasher-) [reply exchanges/bybit/futures_type.go:152]: @shazbert just shed a tear of joy

Comment (shazbert) [reply exchanges/bybit/futures_type.go:152]: 🦜 I approve this comment.

**Review by gloriousCode — APPROVED** [review]
tACK! Thanks for making that change :tada: :parrot:

**Review by shazbert — APPROVED** [review]
Changes ACK!

**Pull request merged**